js/Heap.js

```diff
@@ -1,6 +1,8 @@
 // Libraries
 import { NativeModules, Platform } from 'react-native';
 import Package from 'react-native-package';
+import { extractProps } from './util/extractProps';
```

Comment:
Nit: vertical whitespace between third-party imports and first-party imports

Reply:
done